packages/onboarding/src/utils/test/domain-suggester.ts

```diff
@@ -72,4 +72,16 @@ describe( 'suggestEmailCorrection', () => {
 			wasCorrected: false,
 		} );
 	} );
+
+	test( 'should not suggest a correction for a domain if user is still typing', () => {
+		const result = suggestEmailCorrection( 'example@mail' );
+		expect( result ).toEqual( {
+			oldEmail: 'example@mail',
+			oldDomain: 'mail',
+			newDomain: null,
+			newEmail: null,
+			distance: Infinity,
+			wasCorrected: false,
+		} );
+	} );
 } );
```

Comment:
This test is not great because the distance is 3 and we wouldn't have suggested something anyways. It should be something with distance=1 and a part of a domain that we'd suggest. Sth like `hello@gmail.co`.